gptel-transient.el

```diff
@@ -806,7 +821,8 @@ Or in an extended conversation:
              :position position
              :in-place (and in-place (not output-to-other-buffer-p))
              :stream stream
-             :system (concat gptel--system-message system-extra)
+             :system (when gptel--system-message
```

Comment:
If there's no system message, it should still be possible to add an additional directive from the transient menu (the `d` option captured in `system-extra` here).  This will now function as the whole system message.